azure/requirements.txt

```diff
@@ -23,3 +24,4 @@ requests-oauthlib==2.0.0
 six==1.17.0
 typing_extensions==4.12.2
 urllib3==2.2.3
+uuid==1.30
```

Comment:
the uuid library is part of the [python stdlib](https://docs.python.org/3/library/uuid.html). I think this is pointing to some third party library trying to pose as the std lib.

Reply:
yikes good catch